pkg/reconciler/artifactbuild/artifactbuild.go

```diff
@@ -595,3 +606,27 @@ func (r *ReconcileArtifactBuild) createLookupScmInfoTask(ctx context.Context, lo
 		},
 	}, nil
 }
+
+func (r *ReconcileArtifactBuild) handleCommunityDependencies(ctx context.Context, split []string, namespace string, log logr.Logger) error {
```

Comment:
We already had code to create it for the contaminants, so that is not actually part of this pr, this just move the updating of the 'contaminants' field on the DependencyBuild from the Java code to something done by the controller. Once this field is updated the controller will automatically act on it.